index.html

```
@@ -1,649 +1,663 @@
+
+        <div class="code-block">
+          <button class="copy-btn" aria-label="Copy code">Copy</button>
+          <span
+            class="copy-status"
+            aria-live="polite"
+            style="position: absolute; left: -9999px"
+          ></span>
+          <code
+            >import SocialShareButton from 'social-share-button'; function App() { return (
+            &lt;SocialShareButton url="https://your-website.com" title="Check this out!"
+            onShare={(platform) => { console.log('Shared on:', platform); }} /&gt; ); }</code
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)